pkg/subctl/cmd/export.go

```diff
@@ -0,0 +1,102 @@
+/*
+© 2020 Red Hat, Inc. and others.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/lighthouse/pkg/apis/lighthouse.submariner.io/v2alpha1"
+	lighthouseClientset "github.com/submariner-io/lighthouse/pkg/client/clientset/versioned"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	exportCmd = &cobra.Command{
+		Use:   "export",
+		Short: "Exports a service",
+		Long:  "This command simplifies exporting service(s) to other clusters",
```

Comment:
It seems this should be generalized as we may want to export other things.

```suggestion
		Short: "Exports a resource to other clusters",
		Long:  "This command exports a resource so it is accessible to other clusters",
```

Reply:
I considered this but decided not to for now. I don't think we have anything other than services to be exported. We'll add something like `namespace` but even in that case it will be restricted to only services in that namespace. In case we do support something else, we can revisit help text. Thoughts?

Reply:
So  `subctl export --help`  shows:

_This command simplifies exporting service(s) to other clusters_

_Usage:
  subctl export [command]_

_Available Commands_:
  _service     Creates a ServiceExport resource_

_Flags_: …

I know we’re only using `export` for services (right now) and it’s not a big deal but I still think technically the help should be generic and not reflect that of the/a sub-command.  I mean if we’re going to tie export to its sub-command then we might as well do away with the intermediate and go back to `export-service` :)